charts/trustify/values.yaml

```diff
@@ -87,3 +88,19 @@ modules:
       registry: docker.io/library
       version: "17.2"
     importers: {}
+
+  collector:
+    enabled: false
+    replicas: 1
+    image:
+      name: opentelemetry-collector-contrib
```

Comment:
Now that the project is in another organization I understand that it doesn't make much sense to maintain fidelity with [middlestream](https://github.com/os-observability/redhat-opentelemetry-collector) in relation to the version number, does the same apply to changing the collector from `core` to `contrib`?

Reply:
please discard my question :+1: The answer is probably yes. The middlestream is based on `core` so if we start using something from `contrib` that is not present in `core` it will not work downstream but on the other hand we can not be blocked by that so...